control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_types.go

```diff
@@ -53,6 +54,21 @@ type ConsumerGroup struct {
 	Status ConsumerGroupStatus `json:"status,omitempty"`
 }
 
```

Comment:
mind adding some comments on the new added APIs

Reply:
Done, let me know if there are other places you meant.